src/program.js

```diff
@@ -88,11 +88,11 @@ export class Program {
     const cmd = argv._[0];
 
     if (cmd === undefined) {
-      throw new WebExtError('No sub-command was specified in the args');
+      throw new UsageError('No sub-command was specified in the args');
     }
     let runCommand = this.commands[cmd];
     if (!runCommand) {
-      throw new WebExtError(`unknown command: ${cmd}`);
+      throw new UsageError(`unknown command: ${cmd}`);
```

Comment:
while you're in here could you capitalize the first word so it reads "Unknown command: ${cmd}" ? That should be fixed.